docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rocker/r-ver:3.6.3
```

Comment:
Is artifactory what we should be using now for base images?  

We should probably get this image bumped up to the latest R.  The current one (4.0.5) I don't think it should affect anything in the current dashboard pipeline.  4.1 is scheduled for next week though, so could wait for that image I suppose.